public/css/style.css

```
@@ -1,210 +1,218 @@
+
+  color: var(--default-elao-consent-dark);
+  background-color: var(--default-elao-consent-light);
+  border: none;
+  border-radius: 4px;
+  z-index: 2147483647;
 }
 
 .elao-consent a {
-    color: var(--default-elao-consent-primary);
-    font-weight: bold;
-    text-decoration: underline;
+  color: var(--default-elao-consent-primary);
+  font-weight: bold;
+  text-decoration: underline;
 }
 
 .elao-consent button {
-    padding: 7px 25px;
-    color: var(--default-elao-consent-light);
-    font-weight: bold;
-    background: var(--default-elao-consent-primary);
-    border: none;
-    border-radius: 5px;
+  padding: 7px 25px;
+  color: var(--default-elao-consent-light);
+  font-weight: bold;
+  background: var(--default-elao-consent-primary);
+  border: none;
+  border-radius: 5px;
 }
 
 .elao-consent .message {
-    padding: 0 40px;
+  padding: 0 20px;
+  margin-bottom: 12px;
 }
 
 .elao-consent__details summary {
-    margin: 0 40px;
-    padding-right: 30px;
-    display: inline-block;
-    position: relative;
-    color: var(--default-elao-consent-primary);
-    font-weight: bold;
-    text-decoration: underline;
-    list-style: none;
+  margin: 0 20px;
+  padding-right: 30px;
+  display: inline-block;
+  position: relative;
+  color: var(--default-elao-consent-primary);
+  font-weight: bold;
+  text-decoration: underline;
+  list-style: none;
 }
 
 .elao-consent__details summary:before {
-    position: absolute;
-    top: 0;
-    right: 0;
-    font-size: 22px;
-    content: '🍪';
-    line-height: 1;
+  position: absolute;
+  top: 0;
+  right: -10px;
+  font-size: 30px;
+  content: '🍪';
+  line-height: 1;
 }
 
 .elao-consent__details ul {
-    padding: 20px 40px 0;
+  padding: 20px 40px 0;
 }
 
 .elao-consent__details ul {
-    list-style: none;
+  list-style: none;
 }
 
 .elao-consent__details ul li {
-    margin: 0 0 4px;
-    display: flex;
-    align-items: center;
+  margin: 0 0 4px;
+  display: flex;
+  align-items: center;
 }
 
 .elao-consent__details ul li .choice {
-    display: flex;
-    margin-left:  auto;
+  display: flex;
+  margin-left:  auto;
 }
 
 .elao-consent__details[open] {
-    margin-bottom: 0;
+  margin-bottom: 0;
 }
 
 .elao-consent__details[open] + .elao-consent__actions {
-    display: none;
+  display: none;
 }
 
 .elao-consent__actions {
-    margin: 30px 0 0;
-    display: flex;
+  margin: 15px 0 0;
+  display: flex;
 }
 
 .elao-consent__actions button {
-    flex: 1;
-    height: 50px;
-    font-weight: bold;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    border: none;
-    border-radius: 0;
-    transition: background ease-in .2s;
+  flex: 1;
+  height: 50px;
+  font-weight: bold;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: none;
+  border-top: 1px solid #F1F1F1;
+  border-radius: 0;
+  transition: background ease-in .2s;
 }
 
 .elao-consent__actions button:first-of-type {
-    border-radius: 0 0 0 15px;
+  border-radius: 0 0 0 15px;
```

Ici le border-radius devrait suivre celui du toast. 

```suggestion
  border-radius: 0 0 0 4px;
```